dangerzone/conversion/doc_to_pixels_qubes_wrapper.py

```diff
@@ -67,8 +68,13 @@ async def main() -> None:
     with open("/tmp/input_file", "wb") as f:
         f.write(data)
 
-    converter = DocumentToPixels()
-    await converter.convert()
+    try:
+        converter = DocumentToPixels()
+        await converter.convert()
+    except ConversionException as e:
+        sys.exit(e.error_code)
+    except (RuntimeError, TimeoutError, ValueError) as e:
+        sys.exit(1)
```

Comment:
For debugging reasons, we should print the traceback here, shouldn't we? Same goes for the container conversion.

Reply:
Would something like 5853356 work?

Reply:
Should be fixed in 95210e9

Reply:
Actually I reversed this because it `traceback.format_exc(e)` doesn't take an exception as a parameter and it's a bit more complex to use (I should have tested before pushing). Furthermore, this PR didn't really change it so we can change it in a futher PR.